net/dnsdist/files/dnsdist.init

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2015 OpenWrt.org
+START=20
```

Comment:
Any reason why this can't be 99?

Reply:
I've changed it to 99, this was copied from another file as a template.